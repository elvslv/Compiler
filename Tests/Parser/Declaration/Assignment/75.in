type trec = record x: integer; y: real; end;
var a: record x: integer; y: trec; end;
const m = 10;
begin
a.y.x := 12 + m;
end.