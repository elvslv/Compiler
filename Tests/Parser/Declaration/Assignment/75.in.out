type TREC = record 
    var X : INTEGER;
    var Y : REAL;
end;
var A : record 
    var X : INTEGER;
    var Y : TREC;
end;
const M = 10;

:=           
|            
|___.        
|   |        
|   |___.    
|   |   |    
|   |   |___A
|   |   |    
|   |   |___Y
|   |        
|   |___X    
|            
|___+        
    |        
    |___12   
    |        
    |___M    
